k8s/validator.yml

```diff
@@ -33,7 +33,7 @@ spec:
           livenessProbe:
             tcpSocket:
               port: 9825
-            initialDelaySeconds: 5
+            initialDelaySeconds: 120
```

Comment:
id recommend keeping this number very low and instead set `periodSeconds` and `failureThreshold` to something that would last 2 minutes or more